### PR TITLE
OTA-4174: Make an ostree update a default on IP Secondary

### PR DESCRIPTION
**Pull request by mike-sul**
Signed-off-by: Mykhaylo Sul <ext-mykhaylo.sul@here.com>

### CODE REVIEW

File: recipes-test/demo-config/secondary-config.bb [hunk 1]
```diff
@@ -12,14 +12,14 @@ SOTA_HARDWARE_ID ?= "${MACHINE}-sndry"
 SECONDARY_HARDWARE_ID ?= "${SOTA_HARDWARE_ID}"
 
 SRC_URI = "\
-    file://30-fake-pacman.toml \
+    file://30-ostree-pacman.toml \
```
Comment (patrickvacek) [inline recipes-test/demo-config/secondary-config.bb]: What will happen to `30-fake-pacman.toml`? Will it be unused?

Comment (mike-sul) [reply recipes-test/demo-config/secondary-config.bb]: I don't know, it depends on our decision on the file/binary update configuration.

Comment (patrickvacek) [reply recipes-test/demo-config/secondary-config.bb]: We're only testing ostree in CI, and if there is no demo or test for the fake package manager on the secondary, I would prefer to remove the unused file.

Comment (mike-sul) [reply recipes-test/demo-config/secondary-config.bb]: ok

**Comment by patrickvacek** [comment]
Looks fine now, but of course depends on https://github.com/advancedtelematic/aktualizr/pull/1500.

**Pull request merged**